src/__stories__/sheet-story.tsx

```diff
@@ -58,11 +58,13 @@ export const Default: StoryComponent = () => {
                         setOpen(false);
                     }}
                 >
-                    <ResponsiveLayout>
-                        <Box paddingBottom={{mobile: 16, desktop: 40}} paddingTop={{mobile: 0, desktop: 40}}>
-                            <Placeholder />
-                        </Box>
-                    </ResponsiveLayout>
+                    {({modalTitleId}) => (
```

Comment:
This fixes the issue with the wrong paddings in the sheet's story: https://jira.tid.es/browse/WEB-2101